src/main/java/com/minecolonies/coremod/colony/buildings/modules/SimpleCraftingModule.java

```diff
@@ -39,6 +39,12 @@ public List<IRequestResolver<?>> createResolvers()
         return Collections.emptyList();
     }
 
+    @Override
+    public boolean canLearnManyRecipes()
+    {
+        return false;
+    }
+
     @Override
     public boolean canLearnLargeRecipes()
```

Comment:
but you kept the large recipe type here as a separate method, the idea was to get rid of this extra thing, which is "only good for distinguishing 2x2 vs 3x3 crafting"

Reply:
Getting rid of the extra thing is not a useful goal, especially when that conflates two unrelated things.

Reply:
There are different things, there are MODULE_CRAFTING_TYPES which includes LARGE, SMALL, SMELTING, BREWING (everything that we consider crafting to some sorts) and there is the Mojang RECIPE_TYPE which are two different things imo.

Reply:
Oh, I see the confusion.  You're coming at this from the UI side top-down, and 2x2 and 3x3 crafting have different UIs.  I'm coming at it from the recipe side, bottom-up, and they have the same recipe type.  So you see them as different and I see them as the same.

Reply:
But regardless, I think basing it on RecipeType is better than inventing our own type, since it allows for extension to new crafting types by addons or future features without modifying this type (OCP).  And that sees 2x2 and 3x3 crafting as the same, so this should too.

If you really want to get rid of the large crafting check, it could be replaced with an instanceof, since there's only one type that ever sets it to false.  But I don't really think it needs to be removed at all.

Reply:
Yes, in the end, our workers don't really rely on or care about the vanilla recipe system at all (outside from the UI side of things) that's why I wanted to avoid using the vanilla RecipeType (I was about to use it as well but then decided otherwise).

I do understand the concern of extendability though. Instead of enum we can make this an object with some pre-defined static types as well. This way anyone could create a new static object for their type when required.

I just want to avoid mixing it up with the Recipe Type because our workers don't mess with vanilla recipe handling outside of the teacvhing

Reply:
Well, it's also extremely convenient for JEI and recipe auditing that this does use RecipeType.  Though that doesn't preclude using our own type provided that it's convertable to RecipeType.... but then I don't really see the point of not using RecipeType, other than the annoyance of brewing not already being one.